dev/set_matrix.py

```diff
@@ -496,6 +532,9 @@ def expand_config(config):
             if cfg.minimum not in versions:
                 versions.append(cfg.minimum)
 
+            if cfg.requirements and not is_ref:
+                validate_requirements(cfg.requirements, name, category, package_info, versions)
```

Comment:
The ml-package-versions.yml on master has invalid requirements and needs to skip validation.